templates/buildkite-elastic.yml

```diff
@@ -332,6 +332,16 @@ Conditions:
     UseCostAllocationTags:
       "Fn::Equals": [ { Ref: EnableCostAllocationTags }, "true" ]
 
+    HasKeyName:
+      "Fn::Not": [ "Fn::Equals": [ { Ref: KeyName }, "" ] ]
+
+    EnableSshIngress:
+      "Fn::And":
+        - { Condition: HasKeyName }
+        - { Condition : CreateSecurityGroup }
+        # Enable ingress if authorized users (keys) can be specified another way
+        - "Fn::Not": [ "Fn::Equals": [ { Ref: AuthorizedUsersUrl }, "" ] ]
```

Comment:
This probably needs to be in an `Fn::Or`, or am I booleaning wrong?

Reply:
Well that's embarrassing, good catch! I've pushed up a fix